src/bootstrap/server.bootstrap.js

```diff
@@ -38,6 +41,9 @@ export class Server {
     this.app.use("/health", healthRouter);
     this.app.use("/events", eventRouter);
     this.app.use("/user", userRouter);
+    this.app.use("/my", myRouter);
```

Comment:
I see you have here a separate controller for calling user specific routes (in this case, your own, right?).
Wouldn't it be an idea to call the routes in this controller in their specific one and add the user id as a parameter?

e.g the `getMyEvents` route can be called in the `events` controller using the simple `get all events` method, which now could have the userid sent as parameter?